app/models/spot.rb

```diff
@@ -58,13 +58,13 @@ def create_match(spot)
   end
 
   def notify(match)
-    return unless match.second_user_push_token
+    return unless match.second_user_push_token.any?
```

Comment:
`any?` is used for arrays. Have you tested this?
In this case you should something like `present?`

Reply:
Yes, it is an array. That's because the same user may be subscribed to notifications from multiple devices, so in order to notify all devices I set `push_token` as a serializable array at `app/models/user.rb`.

Reply:
You're right. I didn't see it was a serialized field.